### PR TITLE
Feat/nasha breadcrumb

### PR DESCRIPTION
**Pull request by marie-j**
| Question         | Answer
| ---------------- | ---
| Branch?          | `feat/solarsteein`
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | Fix #... 
| License          | BSD 3-Clause

### CODE REVIEW

File: packages/manager/modules/nasha/src/index.js [hunk 1]
```diff
@@ -3,47 +3,69 @@ import '@ovh-ux/manager-core';
...
+        .state('nasha.nasha-add.**', {
+          url: '/new',
+          lazyLoad,
+        })
+        .state('nasha.nasha-unavailable.**', {
+          url: '/unavailable',
           lazyLoad,
         });
+
+      $urlRouterProvider.when(/^\/paas\/nasha/, () => {
```
Comment (varun257) [inline packages/manager/modules/nasha/src/index.js:63]: Is it possible to use `redirectTo` instead ?

Comment (marie-j) [reply packages/manager/modules/nasha/src/index.js:63]: @varun257 A `redirectTo` will only work for root url (if customer target `/paas/nasha`). If I target `/paas/nasha/{serviceName}` the `redirectTo` won't apply except if for each nasha state we create a paas.nasha state that adds a `redirectTo`, it is easier to maintain the redirection that way imo

**Review by varun257 — APPROVED** [review]

File: packages/manager/modules/nasha/src/dashboard/routing.js [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+      controllerAs: 'NashaOrderCompleteCtrl',
+      translations: {
+        value: ['order', '..'],
+        format: 'json',
+      },
+      params: {
+        orderUrl: null,
+      },
+      resolve: {
+        hideBreadcrumb: /* @ngInject */ () => true,
```
Comment (JeremyDec) [inline packages/manager/modules/nasha/src/dashboard/routing.js:37]: Same thing for the two below
```suggestion
        hideBreadcrumb: () => true,
```

**Review by JeremyDec — APPROVED** [review]

**Comment by marie-j** [comment]
Only updated with @JeremyDec suggestion

**Pull request merged**